spec/acceptance/api/v1/greenhouses_controller_spec.rb

```diff
@@ -53,11 +53,11 @@
     end
   end
 
-  get '/api/v1/greenhouses/:id' do
+  get '/api/v1/buildings/:building_id/greenhouses/:id' do
     example 'Get a greenhouse' do
       authentication :basic, "Bearer #{user_token.token}"
 
-      do_request
+      do_request(building_id: building_id, id: id)
```

Comment:
la méthode `do_request` (ici ou ailleurs dans ce fichier) ne prend pas d'arguments
```suggestion
      do_request
```